node-group-project/src/components/PetTypeContainer.js

```diff
@@ -6,11 +6,11 @@ const PetTypeContainer = (props) => {
 
   let petType = props.match.params.petType;
 
-  // useEffect(() => {
-  //   fetch(`http://localhost:3000/pets/${petType}`)
-  //     .then(result => setAdoptablePets(result.rows))
-  //     .catch(error => console.log(error))
-  // }, []);
+  useEffect(() => {
+    fetch(`http://localhost:3000/pets/${petType}`)
```

Comment:
Much the same as the last comment that you changed. Is the endpoint path going to be listening for that path?